src/Headers/InlineFuncs.h

```diff
@@ -493,10 +456,10 @@ static inline bool ParticleBoxOverlap (SphParticle<ndim>& part, Box<ndim>& box)
 //  ..
 //=================================================================================================
 template <int ndim, template<int> class Particle >
-static inline bool ParticleInBox (const Particle<ndim>& part,const Box<ndim>& box)
+inline bool ParticleInBox (const Particle<ndim>& part,const Box<ndim>& box)
```

Comment:
On a different note, why are these functions static? Static means that each translation unit (i.e., cpp file) gets its own version. But we don't care ( a), they will be inlined anyway, and b) if they arent', we are just producing code bloat). In c++, inline is already enough to prevent having problems with multiple definitions... I think all of these functions should just be inline, without static